articles/connections/passwordless/reference/troubleshoot.md

```diff
@@ -41,4 +37,8 @@ If a user does not receive the requested email or SMS message, here are some thi
 
 ## What if a user receives multiple authentication emails/SMS messages?
 
-If a user accidentally requests more than one email or SMS message containing magic links or authentication codes, the last **five** codes will be valid. After successful log in using any of the codes/links, **all** of the existing codes/links will be invalidated.
\ No newline at end of file
+If a user accidentally requests more than one email or SMS message containing magic links or authentication codes, only the most recent code/link will be valid. After successful log in using any of the codes/links, **all** of the existing codes/links will be invalidated.
+
```

Comment:
> After successful log in using any of the codes/links, **all** of the existing codes/links will be invalidated.

This is not accurate. We are keeping only one code at time, so after successful log in, the only existing code will be invalidated.

Reply:
same conditions you wrote in `/articles/connections/_otp-limitations.md`, right?

Reply:
OK, just removed this conversation and clarified it in the limitations one